config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -467,6 +498,38 @@ periodics:
           cpu: "2"
           memory: "3Gi"
 
+- name: ci-etcd-integration-1-cpu-ppc64le
```

Comment:
question: should we add nodeSelector? I see some job have it, but some not.

Reply:
Hi @ahrtr, currently, the `k8s-infra-ppc64le-prow-build` cluster consists of ppc64le nodes only. I've removed the `nodeSelector` section for uniformity for the jobs that run on `k8s-infra-ppc64le-prow-build`.

Reply:
thx for the clarification.